maths/power_of_two.dart

```diff
@@ -0,0 +1,15 @@
+void power_of_two(int n) {
+  int result = n & (n - 1);
+  if (result == 0)
+    print("Yes, the number $n is a power of 2");
+  else
+    print("No, the number $n is not a power of 2");
+}
+
+void main() {
+  power_of_two(10);
+  power_of_two(23);
+  power_of_two(32);
+  power_of_two(2234);
+  power_of_two(2048);
```

Comment:
Hmm, it's returning `true` for `0`. 
```suggestion
  test("Test power_of_two 0 returns false", () {
    expect(power_of_two(0), isFalse);
  });

  test("Test power_of_two 1 returns true", () {
    expect(power_of_two(1), isTrue);
  });

  test("Test power_of_two 10 returns false", () {
    expect(power_of_two(10), isFalse);
  });

  test("Test power_of_two 10 returns false", () {
    expect(power_of_two(10), isFalse);
  });

  test("Test power_of_two 23 returns false", () {
    expect(power_of_two(23), isFalse);
  });

  test("Test power_of_two 32 returns true", () {
    expect(power_of_two(32), isTrue);
  });

  test("Test power_of_two 2234 returns false", () {
    expect(power_of_two(2234), isFalse);
  });

  test("Test power_of_two 2048 returns true", () {
    expect(power_of_two(2048), isTrue);
  });
```